Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -120,23 +128,29 @@ public void driveWithIMU(double xPower, double yPower, double tPower)  {
         }
 
         // calculate speed and direction of each individual motor and set power of motors to speed
-        motorFL.setPower((-yPower + xPower + tPower) * Constants.DRIVE_SPEED_MULTIPLIER);
-        motorFR.setPower((-yPower - xPower - tPower) * Constants.DRIVE_SPEED_MULTIPLIER);
-        motorBL.setPower((-yPower - xPower + tPower) * Constants.DRIVE_SPEED_MULTIPLIER);
-        motorBR.setPower((-yPower + xPower - tPower) * Constants.DRIVE_SPEED_MULTIPLIER);
+        motorFL.setPower(-yPower + xPower + tPower);
+        motorFR.setPower(-yPower - xPower - tPower);
+        motorBL.setPower(-yPower - xPower + tPower);
+        motorBR.setPower(-yPower + xPower - tPower);
     }
 
-    // method to open/close grabber if you set boolean to true or false
-    public void openGrabber(boolean open) {
-        if (open) {
-            //servoGrabber.setPosition(0.33); // set servo to open position
+    // this method will allow the grabber to open or close given a boolean input, with true = open and false = close
+    public void driveGrabber(boolean isOpen) {
+        if (isOpen) {
+            servoGrabber.setPosition(Constants.GRABBER_OPEN_POSITION); // set servo to open position
         } else {
-            //servoGrabber.setPosition(0.11); // set servo to closed position
+            servoGrabber.setPosition(Constants.GRABBER_CLOSE_POSITION); // set servo to closed position
         }
     }
 
-    // this is a general method to turn the turntable
-    public void rotateTurntable(double power) {
-        //motorTurntable.setPower(power);
+    // this method will allow the slides to move upwards, downwards, outwards, and inwards given a specified x target position and y target position
+    public void driveSlides(/*int xTargetPosition,*/ int yTargetPosition) {
```

Comment:
No commented out code

Reply:
We plan to add xTargetPosition for horizontal slides in the future